src/drive/web/modules/filelist/cells/FileName.jsx

```diff
@@ -73,6 +79,19 @@ const FileName = ({
                 {`${formattedUpdatedAt}${
                   formattedSize ? ` - ${formattedSize}` : ''
                 }`}
+                {(isCarbonCopy || isElectronicSafe) && ' - '}
+                {isCarbonCopy && (
+                  <Icon
+                    icon={CarbonCopyIcon}
+                    className={`u-mr-half ${styles['fil-file-infos--icon']}`}
```

Comment:
I think it is not really necessary to ahve a custom class here ? size={8} ?

After discussion, can we have an issue on the problem that AppIcon does not support size ?

Reply:
https://github.com/cozy/cozy-ui/issues/1688